wagtail/admin/static_src/wagtailadmin/js/page-chooser-modal.js

```diff
@@ -75,7 +75,7 @@ PAGE_CHOOSER_MODAL_ONLOAD_HANDLERS = {
             /* pagination links within search results should be AJAX-fetched
             and the result loaded into .page-results (and ajaxified) */
             $('.page-results a.navigate-pages', modal.body).on('click', function() {
-                $('.page-results', modal.body).load(this.href, ajaxifySearchResults);
```

Comment:
Deleting this line was a red flag for me, but I couldn't find where it was ever used before

Reply:
The significance of `ajaxifySearchResults` is also lost on me. As far as I can see the regression in this PR doesn’t come from this. Might need further confirmation though.